dev-python/psutil/psutil-5.6.6.ebuild

```diff
@@ -0,0 +1,49 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+PYTHON_COMPAT=( python{2_7,3_{6,7,8}} pypy3 )
+
+inherit distutils-r1
+
+DESCRIPTION="Retrieve information on running processes and system utilization"
+HOMEPAGE="https://github.com/giampaolo/psutil https://pypi.org/project/psutil/"
+SRC_URI="mirror://pypi/${PN:0:1}/${PN}/${P}.tar.gz"
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~alpha amd64 arm arm64 hppa ~ia64 ppc ppc64 s390 sparc x86 ~amd64-linux ~x86-linux ~ppc-macos ~x64-macos ~x86-macos ~sparc-solaris ~sparc64-solaris ~x64-solaris ~x86-solaris"
```

Comment:
sorry your frustrated, If your wondering why psutil-5.6.6 looks this way its because its an identical copy of psutil-5.7.0, WHICH I DID NOT WRITE! and has been in tree for some time now.

I am also not thrilled about having to fix a bunch of python deps that I don't actually care about. I'm taking ownership of to fix some other packages.

Lack of maintainer-ship is Gentoo is not an easy thing, so we are all trying to contribute where we can.

Thanks for being patient